### ISSUE TITLE
Failed to install on a fresh Symfony 5 install

### ISSUE THREAD

**Issue by gulien** [thecodingmachine/tdbm-bundle#5]
Hello,

I cannot install this bundle on a fresh Symfony 5 install :

```bash
$ composer require thecodingmachine/tdbm-bundle
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Restricting packages listed in "symfony/symfony" to "5.0.*"
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - thecodingmachine/tdbm v5.1.2 requires symfony/console ^2 || ^3 || ^4 -> no matching package found.
    - thecodingmachine/tdbm v5.1.0 requires symfony/console ^2 || ^3 || ^4 -> no matching package found.
    - thecodingmachine/tdbm 5.1.1 requires symfony/console ^2 || ^3 || ^4 -> no matching package found.
    - Conclusion: don't install thecodingmachine/tdbm-bundle v5.1.1
    - Conclusion: remove doctrine/doctrine-bundle 2.0.2
    - Conclusion: don't install doctrine/doctrine-bundle 2.0.2
    - twig/twig v2.0.0 conflicts with doctrine/doctrine-bundle[2.0.2].
    - twig/twig v2.1.0 conflicts with doctrine/doctrine-bundle[2.0.2].
    - twig/twig v2.2.0 conflicts with doctrine/doctrine-bundle[2.0.2].
    - twig/twig v2.3.0 conflicts with doctrine/doctrine-bundle[2.0.2].
    - twig/twig v2.3.1 conflicts with doctrine/doctrine-bundle[2.0.2].
    - twig/twig v2.3.2 conflicts with doctrine/doctrine-bundle[2.0.2].
    - Conclusion: don't install symfony/http-kernel v5.0.1|install twig/twig v2.0.0|install twig/twig v2.1.0|install twig/twig v2.2.0|install twig/twig v2.3.0|install twig/twig v2.3.1|install twig/twig v2.3.2
    - Conclusion: remove symfony/http-kernel v5.0.1|install twig/twig v2.0.0|install twig/twig v2.1.0|install twig/twig v2.2.0|install twig/twig v2.3.0|install twig/twig v2.3.1|install twig/twig v2.3.2
    - twig/twig 1.x-dev conflicts with symfony/http-kernel[v5.0.1].
    - twig/twig v1.34.4 conflicts with symfony/http-kernel[v5.0.1].
    - twig/twig v1.35.0 conflicts with symfony/http-kernel[v5.0.1].
    - twig/twig v1.35.1 conflicts with symfony/http-kernel[v5.0.1].
    - twig/twig v1.35.2 conflicts with symfony/http-kernel[v5.0.1].
    - twig/twig v1.35.3 conflicts with symfony/http-kernel[v5.0.1].
    - twig/twig v1.35.4 conflicts with symfony/http-kernel[v5.0.1].
    - twig/twig v1.36.0 conflicts with symfony/http-kernel[v5.0.1].
    - twig/twig v1.37.0 conflicts with symfony/http-kernel[v5.0.1].
    - twig/twig v1.37.1 conflicts with symfony/http-kernel[v5.0.1].
    - twig/twig v1.38.0 conflicts with symfony/http-kernel[v5.0.1].
    - twig/twig v1.38.1 conflicts with symfony/http-kernel[v5.0.1].
    - twig/twig v1.38.2 conflicts with symfony/http-kernel[v5.0.1].
    - twig/twig v1.38.3 conflicts with symfony/http-kernel[v5.0.1].
    - twig/twig v1.38.4 conflicts with symfony/http-kernel[v5.0.1].
    - twig/twig v1.39.0 conflicts with symfony/http-kernel[v5.0.1].
    - twig/twig v1.39.1 conflicts with symfony/http-kernel[v5.0.1].
    - twig/twig v1.40.0 conflicts with symfony/http-kernel[v5.0.1].
    - twig/twig v1.40.1 conflicts with symfony/http-kernel[v5.0.1].
    - twig/twig v1.41.0 conflicts with symfony/http-kernel[v5.0.1].
    - twig/twig v1.42.0 conflicts with symfony/http-kernel[v5.0.1].
    - twig/twig v1.42.1 conflicts with symfony/http-kernel[v5.0.1].
    - twig/twig v1.42.2 conflicts with symfony/http-kernel[v5.0.1].
    - twig/twig v1.42.3 conflicts with symfony/http-kernel[v5.0.1].
    - twig/twig v1.42.4 conflicts with symfony/http-kernel[v5.0.1].
    - Conclusion: don't install twig/twig v3.0.0|install twig/twig 1.x-dev|install twig/twig v1.34.4|install twig/twig v1.35.0|install twig/twig v1.35.1|install twig/twig v1.35.2|install twig/twig v1.35.3|install twig/twig v1.35.4|install twig/twig v1.36.0|install twig/twig v1.37.0|install twig/twig v1.37.1|install twig/twig v1.38.0|install twig/twig v1.38.1|install twig/twig v1.38.2|install twig/twig v1.38.3|install twig/twig v1.38.4|install twig/twig v1.39.0|install twig/twig v1.39.1|install twig/twig v1.40.0|install twig/twig v1.40.1|install twig/twig v1.41.0|install twig/twig v1.42.0|install twig/twig v1.42.1|install twig/twig v1.42.2|install twig/twig v1.42.3|install twig/twig v1.42.4|install twig/twig v2.0.0|install twig/twig v2.1.0|install twig/twig v2.2.0|install twig/twig v2.3.0|install twig/twig v2.3.1|install twig/twig v2.3.2
    - Conclusion: don't install thecodingmachine/tdbm-bundle v5.1.1|remove twig/twig v3.0.0|install twig/twig 1.x-dev|install twig/twig v1.34.4|install twig/twig v1.35.0|install twig/twig v1.35.1|install twig/twig v1.35.2|install twig/twig v1.35.3|install twig/twig v1.35.4|install twig/twig v1.36.0|install twig/twig v1.37.0|install twig/twig v1.37.1|install twig/twig v1.38.0|install twig/twig v1.38.1|install twig/twig v1.38.2|install twig/twig v1.38.3|install twig/twig v1.38.4|install twig/twig v1.39.0|install twig/twig v1.39.1|install twig/twig v1.40.0|install twig/twig v1.40.1|install twig/twig v1.41.0|install twig/twig v1.42.0|install twig/twig v1.42.1|install twig/twig v1.42.2|install twig/twig v1.42.3|install twig/twig v1.42.4|install twig/twig v2.0.0|install twig/twig v2.1.0|install twig/twig v2.2.0|install twig/twig v2.3.0|install twig/twig v2.3.1|install twig/twig v2.3.2
    - Conclusion: don't install thecodingmachine/tdbm-bundle v5.1.1|don't install twig/twig v3.0.0|install twig/twig 1.x-dev|install twig/twig v1.34.4|install twig/twig v1.35.0|install twig/twig v1.35.1|install twig/twig v1.35.2|install twig/twig v1.35.3|install twig/twig v1.35.4|install twig/twig v1.36.0|install twig/twig v1.37.0|install twig/twig v1.37.1|install twig/twig v1.38.0|install twig/twig v1.38.1|install twig/twig v1.38.2|install twig/twig v1.38.3|install twig/twig v1.38.4|install twig/twig v1.39.0|install twig/twig v1.39.1|install twig/twig v1.40.0|install twig/twig v1.40.1|install twig/twig v1.41.0|install twig/twig v1.42.0|install twig/twig v1.42.1|install twig/twig v1.42.2|install twig/twig v1.42.3|install twig/twig v1.42.4|install twig/twig v2.0.0|install twig/twig v2.1.0|install twig/twig v2.2.0|install twig/twig v2.3.0|install twig/twig v2.3.1|install twig/twig v2.3.2
    - Conclusion: don't install thecodingmachine/tdbm-bundle v5.1.1|remove twig/twig v3.0.0|install twig/twig 1.x-dev|install twig/twig v1.34.4|install twig/twig v1.35.0|install twig/twig v1.35.1|install twig/twig v1.35.2|install twig/twig v1.35.3|install twig/twig v1.35.4|install twig/twig v1.36.0|install twig/twig v1.37.0|install twig/twig v1.37.1|install twig/twig v1.38.0|install twig/twig v1.38.1|install twig/twig v1.38.2|install twig/twig v1.38.3|install twig/twig v1.38.4|install twig/twig v1.39.0|install twig/twig v1.39.1|install twig/twig v1.40.0|install twig/twig v1.40.1|install twig/twig v1.41.0|install twig/twig v1.42.0|install twig/twig v1.42.1|install twig/twig v1.42.2|install twig/twig v1.42.3|install twig/twig v1.42.4|install twig/twig v2.0.0|install twig/twig v2.1.0|install twig/twig v2.2.0|install twig/twig v2.3.0|install twig/twig v2.3.1|install twig/twig v2.3.2
    - Conclusion: don't install thecodingmachine/tdbm-bundle v5.1.1|don't install twig/twig v3.0.0|install twig/twig 1.x-dev|install twig/twig v1.34.4|install twig/twig v1.35.0|install twig/twig v1.35.1|install twig/twig v1.35.2|install twig/twig v1.35.3|install twig/twig v1.35.4|install twig/twig v1.36.0|install twig/twig v1.37.0|install twig/twig v1.37.1|install twig/twig v1.38.0|install twig/twig v1.38.1|install twig/twig v1.38.2|install twig/twig v1.38.3|install twig/twig v1.38.4|install twig/twig v1.39.0|install twig/twig v1.39.1|install twig/twig v1.40.0|install twig/twig v1.40.1|install twig/twig v1.41.0|install twig/twig v1.42.0|install twig/twig v1.42.1|install twig/twig v1.42.2|install twig/twig v1.42.3|install twig/twig v1.42.4|install twig/twig v2.0.0|install twig/twig v2.1.0|install twig/twig v2.2.0|install twig/twig v2.3.0|install twig/twig v2.3.1|install twig/twig v2.3.2
    - Conclusion: don't install thecodingmachine/tdbm-bundle v5.1.1|remove symfony/http-kernel v5.0.1|install twig/twig v2.0.0|install twig/twig v2.1.0|install twig/twig v2.2.0|install twig/twig v2.3.0|install twig/twig v2.3.1|install twig/twig v2.3.2
    - Conclusion: don't install thecodingmachine/tdbm-bundle v5.1.1|don't install symfony/http-kernel v5.0.1|install twig/twig v2.0.0|install twig/twig v2.1.0|install twig/twig v2.2.0|install twig/twig v2.3.0|install twig/twig v2.3.1|install twig/twig v2.3.2
    - Conclusion: don't install thecodingmachine/tdbm-bundle v5.1.1|remove twig/twig v3.0.0|install twig/twig 1.x-dev|install twig/twig v1.34.4|install twig/twig v1.35.0|install twig/twig v1.35.1|install twig/twig v1.35.2|install twig/twig v1.35.3|install twig/twig v1.35.4|install twig/twig v1.36.0|install twig/twig v1.37.0|install twig/twig v1.37.1|install twig/twig v1.38.0|install twig/twig v1.38.1|install twig/twig v1.38.2|install twig/twig v1.38.3|install twig/twig v1.38.4|install twig/twig v1.39.0|install twig/twig v1.39.1|install twig/twig v1.40.0|install twig/twig v1.40.1|install twig/twig v1.41.0|install twig/twig v1.42.0|install twig/twig v1.42.1|install twig/twig v1.42.2|install twig/twig v1.42.3|install twig/twig v1.42.4|install twig/twig v2.0.0|install twig/twig v2.1.0|install twig/twig v2.2.0|install twig/twig v2.3.0|install twig/twig v2.3.1|install twig/twig v2.3.2
    - Conclusion: don't install thecodingmachine/tdbm-bundle v5.1.1|don't install twig/twig v3.0.0|install twig/twig 1.x-dev|install twig/twig v1.34.4|install twig/twig v1.35.0|install twig/twig v1.35.1|install twig/twig v1.35.2|install twig/twig v1.35.3|install twig/twig v1.35.4|install twig/twig v1.36.0|install twig/twig v1.37.0|install twig/twig v1.37.1|install twig/twig v1.38.0|install twig/twig v1.38.1|install twig/twig v1.38.2|install twig/twig v1.38.3|install twig/twig v1.38.4|install twig/twig v1.39.0|install twig/twig v1.39.1|install twig/twig v1.40.0|install twig/twig v1.40.1|install twig/twig v1.41.0|install twig/twig v1.42.0|install twig/twig v1.42.1|install twig/twig v1.42.2|install twig/twig v1.42.3|install twig/twig v1.42.4|install twig/twig v2.0.0|install twig/twig v2.1.0|install twig/twig v2.2.0|install twig/twig v2.3.0|install twig/twig v2.3.1|install twig/twig v2.3.2
    - Installation request for doctrine/doctrine-bundle (locked at 2.0.2) -> satisfiable by doctrine/doctrine-bundle[2.0.2].
    - Conclusion: don't install thecodingmachine/tdbm-bundle v5.1.1|remove symfony/http-kernel v5.0.1|install twig/twig v2.0.0|install twig/twig v2.1.0|install twig/twig v2.2.0|install twig/twig v2.3.0|install twig/twig v2.3.1|install twig/twig v2.3.2
    - Conclusion: don't install thecodingmachine/tdbm-bundle v5.1.1|don't install symfony/http-kernel v5.0.1|install twig/twig v2.0.0|install twig/twig v2.1.0|install twig/twig v2.2.0|install twig/twig v2.3.0|install twig/twig v2.3.1|install twig/twig v2.3.2
    - Installation request for symfony/http-kernel (locked at v5.0.1) -> satisfiable by symfony/http-kernel[v5.0.1].
    - Conclusion: don't install thecodingmachine/tdbm-bundle v5.1.1|remove twig/twig v3.0.0|install twig/twig 1.x-dev|install twig/twig v1.34.4|install twig/twig v1.35.0|install twig/twig v1.35.1|install twig/twig v1.35.2|install twig/twig v1.35.3|install twig/twig v1.35.4|install twig/twig v1.36.0|install twig/twig v1.37.0|install twig/twig v1.37.1|install twig/twig v1.38.0|install twig/twig v1.38.1|install twig/twig v1.38.2|install twig/twig v1.38.3|install twig/twig v1.38.4|install twig/twig v1.39.0|install twig/twig v1.39.1|install twig/twig v1.40.0|install twig/twig v1.40.1|install twig/twig v1.41.0|install twig/twig v1.42.0|install twig/twig v1.42.1|install twig/twig v1.42.2|install twig/twig v1.42.3|install twig/twig v1.42.4|install twig/twig v2.0.0|install twig/twig v2.1.0|install twig/twig v2.2.0|install twig/twig v2.3.0|install twig/twig v2.3.1|install twig/twig v2.3.2
    - Conclusion: don't install thecodingmachine/tdbm-bundle v5.1.1|don't install twig/twig v3.0.0|install twig/twig 1.x-dev|install twig/twig v1.34.4|install twig/twig v1.35.0|install twig/twig v1.35.1|install twig/twig v1.35.2|install twig/twig v1.35.3|install twig/twig v1.35.4|install twig/twig v1.36.0|install twig/twig v1.37.0|install twig/twig v1.37.1|install twig/twig v1.38.0|install twig/twig v1.38.1|install twig/twig v1.38.2|install twig/twig v1.38.3|install twig/twig v1.38.4|install twig/twig v1.39.0|install twig/twig v1.39.1|install twig/twig v1.40.0|install twig/twig v1.40.1|install twig/twig v1.41.0|install twig/twig v1.42.0|install twig/twig v1.42.1|install twig/twig v1.42.2|install twig/twig v1.42.3|install twig/twig v1.42.4|install twig/twig v2.0.0|install twig/twig v2.1.0|install twig/twig v2.2.0|install twig/twig v2.3.0|install twig/twig v2.3.1|install twig/twig v2.3.2
    - Installation request for thecodingmachine/tdbm-bundle ^5.1.1 -> satisfiable by thecodingmachine/tdbm-bundle[5.1.x-dev, v5.1.1].
    - Conclusion: remove twig/twig v3.0.0|install twig/twig 1.x-dev|install twig/twig v1.34.4|install twig/twig v1.35.0|install twig/twig v1.35.1|install twig/twig v1.35.2|install twig/twig v1.35.3|install twig/twig v1.35.4|install twig/twig v1.36.0|install twig/twig v1.37.0|install twig/twig v1.37.1|install twig/twig v1.38.0|install twig/twig v1.38.1|install twig/twig v1.38.2|install twig/twig v1.38.3|install twig/twig v1.38.4|install twig/twig v1.39.0|install twig/twig v1.39.1|install twig/twig v1.40.0|install twig/twig v1.40.1|install twig/twig v1.41.0|install twig/twig v1.42.0|install twig/twig v1.42.1|install twig/twig v1.42.2|install twig/twig v1.42.3|install twig/twig v1.42.4|install twig/twig v2.0.0|install twig/twig v2.1.0|install twig/twig v2.2.0|install twig/twig v2.3.0|install twig/twig v2.3.1|install twig/twig v2.3.2
    - thecodingmachine/tdbm-bundle 5.1.x-dev requires thecodingmachine/tdbm ~5.1.0 -> satisfiable by thecodingmachine/tdbm[5.1.1, 5.1.x-dev, v5.1.0, v5.1.2, v5.1.3].
    - thecodingmachine/tdbm 5.1.x-dev requires mouf/magic-query ^1.3.0 -> satisfiable by mouf/magic-query[1.3.x-dev, 1.4.x-dev, v1.3.0].
    - thecodingmachine/tdbm v5.1.3 requires mouf/magic-query ^1.3.0 -> satisfiable by mouf/magic-query[1.3.x-dev, 1.4.x-dev, v1.3.0].
    - mouf/magic-query v1.3.0 requires twig/twig ^1.34.4 || ^2 -> satisfiable by twig/twig[1.x-dev, 2.x-dev, v1.34.4, v1.35.0, v1.35.1, v1.35.2, v1.35.3, v1.35.4, v1.36.0, v1.37.0, v1.37.1, v1.38.0, v1.38.1, v1.38.2, v1.38.3, v1.38.4, v1.39.0, v1.39.1, v1.40.0, v1.40.1, v1.41.0, v1.42.0, v1.42.1, v1.42.2, v1.42.3, v1.42.4, v2.0.0, v2.1.0, v2.10.0, v2.11.0, v2.11.1, v2.11.2, v2.11.3, v2.12.0, v2.12.1, v2.12.2, v2.2.0, v2.3.0, v2.3.1, v2.3.2, v2.4.0, v2.4.1, v2.4.2, v2.4.3, v2.4.4, v2.4.5, v2.4.6, v2.4.7, v2.4.8, v2.5.0, v2.6.0, v2.6.1, v2.6.2, v2.7.0, v2.7.1, v2.7.2, v2.7.3, v2.7.4, v2.8.0, v2.8.1, v2.9.0].
    - mouf/magic-query 1.4.x-dev requires twig/twig ^1.34.4 || ^2 -> satisfiable by twig/twig[1.x-dev, 2.x-dev, v1.34.4, v1.35.0, v1.35.1, v1.35.2, v1.35.3, v1.35.4, v1.36.0, v1.37.0, v1.37.1, v1.38.0, v1.38.1, v1.38.2, v1.38.3, v1.38.4, v1.39.0, v1.39.1, v1.40.0, v1.40.1, v1.41.0, v1.42.0, v1.42.1, v1.42.2, v1.42.3, v1.42.4, v2.0.0, v2.1.0, v2.10.0, v2.11.0, v2.11.1, v2.11.2, v2.11.3, v2.12.0, v2.12.1, v2.12.2, v2.2.0, v2.3.0, v2.3.1, v2.3.2, v2.4.0, v2.4.1, v2.4.2, v2.4.3, v2.4.4, v2.4.5, v2.4.6, v2.4.7, v2.4.8, v2.5.0, v2.6.0, v2.6.1, v2.6.2, v2.7.0, v2.7.1, v2.7.2, v2.7.3, v2.7.4, v2.8.0, v2.8.1, v2.9.0].
    - mouf/magic-query 1.3.x-dev requires twig/twig ^1.34.4 || ^2 -> satisfiable by twig/twig[1.x-dev, 2.x-dev, v1.34.4, v1.35.0, v1.35.1, v1.35.2, v1.35.3, v1.35.4, v1.36.0, v1.37.0, v1.37.1, v1.38.0, v1.38.1, v1.38.2, v1.38.3, v1.38.4, v1.39.0, v1.39.1, v1.40.0, v1.40.1, v1.41.0, v1.42.0, v1.42.1, v1.42.2, v1.42.3, v1.42.4, v2.0.0, v2.1.0, v2.10.0, v2.11.0, v2.11.1, v2.11.2, v2.11.3, v2.12.0, v2.12.1, v2.12.2, v2.2.0, v2.3.0, v2.3.1, v2.3.2, v2.4.0, v2.4.1, v2.4.2, v2.4.3, v2.4.4, v2.4.5, v2.4.6, v2.4.7, v2.4.8, v2.5.0, v2.6.0, v2.6.1, v2.6.2, v2.7.0, v2.7.1, v2.7.2, v2.7.3, v2.7.4, v2.8.0, v2.8.1, v2.9.0].
    - Can only install one of: twig/twig[2.x-dev, v3.0.0].
    - Can only install one of: twig/twig[v2.10.0, v3.0.0].
    - Can only install one of: twig/twig[v2.11.0, v3.0.0].
    - Can only install one of: twig/twig[v2.11.1, v3.0.0].
    - Can only install one of: twig/twig[v2.11.2, v3.0.0].
    - Can only install one of: twig/twig[v2.11.3, v3.0.0].
    - Can only install one of: twig/twig[v2.12.0, v3.0.0].
    - Can only install one of: twig/twig[v2.12.1, v3.0.0].
    - Can only install one of: twig/twig[v2.12.2, v3.0.0].
    - Can only install one of: twig/twig[v2.4.0, v3.0.0].
    - Can only install one of: twig/twig[v2.4.1, v3.0.0].
    - Can only install one of: twig/twig[v2.4.2, v3.0.0].
    - Can only install one of: twig/twig[v2.4.3, v3.0.0].
    - Can only install one of: twig/twig[v2.4.4, v3.0.0].
    - Can only install one of: twig/twig[v2.4.5, v3.0.0].
    - Can only install one of: twig/twig[v2.4.6, v3.0.0].
    - Can only install one of: twig/twig[v2.4.7, v3.0.0].
    - Can only install one of: twig/twig[v2.4.8, v3.0.0].
    - Can only install one of: twig/twig[v2.5.0, v3.0.0].
    - Can only install one of: twig/twig[v2.6.0, v3.0.0].
    - Can only install one of: twig/twig[v2.6.1, v3.0.0].
    - Can only install one of: twig/twig[v2.6.2, v3.0.0].
    - Can only install one of: twig/twig[v2.7.0, v3.0.0].
    - Can only install one of: twig/twig[v2.7.1, v3.0.0].
    - Can only install one of: twig/twig[v2.7.2, v3.0.0].
    - Can only install one of: twig/twig[v2.7.3, v3.0.0].
    - Can only install one of: twig/twig[v2.7.4, v3.0.0].
    - Can only install one of: twig/twig[v2.8.0, v3.0.0].
    - Can only install one of: twig/twig[v2.8.1, v3.0.0].
    - Can only install one of: twig/twig[v2.9.0, v3.0.0].
    - Installation request for twig/twig (locked at v3.0.0) -> satisfiable by twig/twig[v3.0.0].
```

Here is my `composer.json`:

```json
{
    "type": "project",
    "license": "proprietary",
    "require": {
        "php": "^7.2.5",
        "ext-ctype": "*",
        "ext-iconv": "*",
        "bref/bref": "dev-master",
        "sensio/framework-extra-bundle": "^5.1",
        "symfony/asset": "5.0.*",
        "symfony/console": "5.0.*",
        "symfony/dotenv": "5.0.*",
        "symfony/expression-language": "5.0.*",
        "symfony/flex": "^1.3.1",
        "symfony/form": "5.0.*",
        "symfony/framework-bundle": "5.0.*",
        "symfony/http-client": "5.0.*",
        "symfony/intl": "5.0.*",
        "symfony/mailer": "5.0.*",
        "symfony/monolog-bundle": "^3.1",
        "symfony/notifier": "5.0.*",
        "symfony/orm-pack": "*",
        "symfony/process": "5.0.*",
        "symfony/security-bundle": "5.0.*",
        "symfony/serializer-pack": "*",
        "symfony/string": "5.0.*",
        "symfony/translation": "5.0.*",
        "symfony/twig-pack": "*",
        "symfony/validator": "5.0.*",
        "symfony/web-link": "5.0.*",
        "symfony/yaml": "5.0.*"
    },
    "require-dev": {
        "symfony/debug-pack": "*",
        "symfony/maker-bundle": "^1.0",
        "symfony/profiler-pack": "*",
        "symfony/test-pack": "*"
    },
    "config": {
        "preferred-install": {
            "*": "dist"
        },
        "sort-packages": true
    },
    "autoload": {
        "psr-4": {
            "App\\": "src/"
        }
    },
    "autoload-dev": {
        "psr-4": {
            "App\\Tests\\": "tests/"
        }
    },
    "replace": {
        "paragonie/random_compat": "2.*",
        "symfony/polyfill-ctype": "*",
        "symfony/polyfill-iconv": "*",
        "symfony/polyfill-php72": "*",
        "symfony/polyfill-php71": "*",
        "symfony/polyfill-php70": "*",
        "symfony/polyfill-php56": "*"
    },
    "scripts": {
        "auto-scripts": {
            "cache:clear": "symfony-cmd",
            "assets:install %PUBLIC_DIR%": "symfony-cmd"
        },
        "post-install-cmd": [
            "@auto-scripts"
        ],
        "post-update-cmd": [
            "@auto-scripts"
        ]
    },
    "conflict": {
        "symfony/symfony": "*"
    },
    "extra": {
        "symfony": {
            "allow-contrib": false,
            "require": "5.0.*"
        }
    },
    "minimum-stability": "dev",
    "prefer-stable": true
}
```

**Comment by moufmouf** [comment]
Could you try forcing TDBM to ^5.2?

```
composer require thecodingmachine/tdbm-bundle ^5.2
```

**Comment by gulien** [comment]
Did you mean `tdbm` instead of `tdbm-bundle` 😄?

If so:

```bash
$ composer require thecodingmachine/tdbm "5.2.x-dev"
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Restricting packages listed in "symfony/symfony" to "5.0.*"
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - brain-diminished/schema-version-control 1.x-dev requires symfony/yaml ^3.3 -> no matching package found.
    - brain-diminished/schema-version-control 1.0.2 requires symfony/yaml ^4.1 -> no matching package found.
    - thecodingmachine/tdbm 5.2.x-dev requires brain-diminished/schema-version-control ^1.0.2 -> satisfiable by brain-diminished/schema-version-control[1.0.2, 1.x-dev].
    - Installation request for thecodingmachine/tdbm 5.2.x-dev -> satisfiable by thecodingmachine/tdbm[5.2.x-dev].

```

**Comment by moufmouf** [comment]
Damn'it!
:boom:

**Comment by kevinsechocho** [comment]
Hi Gulien,

I had the same problem with "knplabs/knp-paginator-bundle" when upgrading from version 4.4 to 5.0.

Thus, "composer require thecodingmachine/tdbm" requires "symfony/console" version 4.

Solution: in your composer.json change all symfony/* version "^4.4" to "^4.*|^5.0"

        "symfony/asset": "^4.4|^5.0",
        "symfony/console": "^4.4|^5.0",
        "symfony/expression-language": "^4.4|^5.0",
        "symfony/framework-bundle": "^4.4|^5.0",
        "symfony/http-client": "^4.4|^5.0",
        "symfony/intl": "^4.4|^5.0",
        "symfony/mailer": "^4.4|^5.0",
        "symfony/notifier": "^4.4|^5.0",
        "symfony/security-bundle": "^4.4|^5.0",
                  ............etc.............

And the lastly

    "extra": {
        "symfony": {
            "allow-contrib": false,
            "require": "4.4.*"
        }
    },
 
That's it.

**Comment by gulien** [comment]
Resolved with the release of magic query 1.4 :tada:
